README.md

```diff
@@ -15,20 +15,56 @@
 |:----------------:|:----------------------------:|
 | v0.1.*           | v1.5.*                       |
 
-## Installation
+## Using this starter kit
 
-Add the following dependency to your Akeneo PIM project with this shell command:
+### Context
+To write an extension, you will need to integrate your extension in an existing PIM installation.
+We will use the PIM Community Standard Edition as the host of our connector.
 
+### Initialisation
+Create a new host project with the Standard Edition
+
+```
+composer create-project --prefer-dist akeneo/pim-community-standard /home/myhome/pim-project "1.5.*@stable"
+cd /home/myhome/pim-project
+```
+
+This will copy a copy of the standard edition without git informations.
+
+Then, clone the starter kit
+
+```
+composer require akeneo-labs/extension-starter 0.1.0 --prefer-dist
 ```
-composer require "akeneo-labs/extension-starter" "0.1.0";
+
+### Customization
+You will need to rename your extension according to the chosen name.
+Let's say you work for the WorldCompany and the connector name is WorldConnector.
+
+You will need to move your component inside the vendor directory according to your company name:
+
+```
+cd vendor
+mkdir world-company
+mv akeneo-labs/extension-starter world-company/world-connector
+cd world-company/world-connector
+mv Acme WorldCompany
 ```
 
-Register your new bundle in your AppKernel.php,
+And then change the namespaces:
+
+```
+find WorldCompany/ -name '*.php' -o -name 'composer.json' -type f -print0 | xargs -0 sed -i 's#Acme#WorldCompany#g'
+cd WorldCompany/Bundle/DemoExtensionBundle/
+cp doc/composer.json.dist composer.json
+sed -i 's#Acme#WorldCompany#g' composer.json
+sed -i 's#acme#world-company#g' composer.json
+```
+And finally, clean up all the starter kit initialisation files:
```

Comment:
Initiali**z**ation (you wrote cusotmi**z**ation, you have to stay consistent between UK/US English)